README.md

```diff
@@ -236,6 +237,7 @@ Name | Website | Region
 [Fire Engine Red](/company-profiles/fire-engine-red.md) | https://fire-engine-red.com/ | Worldwide
 [Fireball Labs](/company-profiles/fireball-labs.md) | https://www.fireballlabs.com | Germany
 [FivexL](/company-profiles/fivexl.md) | https://fivexl.io | Worldwide
+[Fiverr](/company-profiles/fiverr.md) | https://www.fiverr.com/ | North America, Asia
```

Comment:
Please make another pull request for `fiverr`

Reply:
okay